components/basecomponents/AlphabetCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import PlayIcon from '../../public/icons/play.svg';
+
+interface AlphabetCardProps {
+  examples: {
+    example: string;
+    example_transcription: string;
+  }[];
+  player: HTMLAudioElement | null;
+  setPlayer: React.Dispatch<React.SetStateAction<HTMLAudioElement | null>>;
+  letter: (string | null)[];
```

Comment:
Maybe this should be `letters: [string, string | null]`

Reply:
Since I moved alphabet to JSON format it seems that typescript is not able to infer tuple and instead infers `(string | null)[]`. But I can be mistaken.

Reply:
I see, yeah. It's not a big deal, but maybe it would be better to store alphabets in `.ts` files instead of pure JSON? We could explicitly define the `Letter` interface and export an array like `ALPHABET_CZ: Letter[]`. You would get more precise types, and also TypeScript would prevent you from accidentally changing the alphabet in unexpected ways. I can make the change if you agree :)

Reply:
Sure we can changed it.